src/pages/Preferences.vue

```diff
@@ -20,13 +23,14 @@ export default Vue.extend({
   layout: 'preferences',
   data() {
     return {
-      currentNavItem:    'Application',
+      currentNavItem:    this.$store.getters['transientSettings/getPreferences'].currentNavItem.name,
```

Comment:
We should be able to treat `currentNavItem` as a computed property and remove the need for the `Preferences` page to track the state of `currentNavItem` entirely. 

I recommend that we refactor this PR in the following way:

1. Add a new `getCurrentNavItem` getter to `store/transientSettings`
2. Remove this `currentNavItem` data prop
3. Add a new `mapGetter` for `transientSettings/getCurrentNavItem` to this page
4. Replace all instances of `currentNavItem` with `getCurrentNavItem` where appropriate (there's an assignment for `currentNavItem` further down that can be removed entirely)

Doing this will allow us to take advantage of the existing reactivity of our vuex state and manage state for the `currentNavItem` in a single location (vuex).

Reply:
You are right, this way the code is much more clean. I made a similar refactor also in `BodyApplication.vue` for `activeTab` variable. Thanks Phil.